progenitor-impl/src/method.rs

```diff
@@ -734,6 +765,45 @@ impl Generator {
             (query_build, query_use)
         };
 
+        let headers = method
+            .params
+            .iter()
+            .filter_map(|param| match &param.kind {
+                OperationParameterKind::Header(required) => {
+                    let qn = &param.api_name;
+                    let qn_ident = format_ident!("{}", &param.name);
```

Comment:
I think this stood for "query name", but this is fine...

Reply:
Oh, yeah. Changed to `hn` to keep the naming style